### DISCUSSION TITLE
Consider including CodeChecker to CDT-Cloud

### DISCUSSION THREAD

**Discussion by MatthewKhouzam** [eclipsesource/cdt-cloud#7] [Ideas]
Code Checker (https://github.com/Ericsson/CodeChecker) is  a static analyzer that uses clang-
tidy but also performs data flow and security checks on code. It could be an item to watch.

soon a vs code plug-in will be available:
https://github.com/Ericsson/CodeCheckerVSCodePlugin/releases

**Comment by JonasHelming** [comment]
@planger Do we already have a place in the official projects for suggestions like this?

**Comment by planger** [comment]
Yes, since the day before yesterday, we have now https://github.com/eclipse-cdt-cloud/cdt-cloud.
I suggest to add all suggestions to https://github.com/eclipse-cdt-cloud/cdt-cloud/discussions/categories/ideas.
Thanks!

**Reply by MatthewKhouzam** [reply]
Updating my bookmarks.
